Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -0,0 +1,880 @@
+                }
+            }
+        });
+        /**
+         * Gets or sets the flag to show content volumes.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.showContentBoundingVolumes = false;
+
+        var showRequestVolumes = knockout.observable();
+        knockout.defineProperty(this, 'showRequestVolumes', {
+            get : function() {
+                return showRequestVolumes();
+            },
+            set : function(value) {
+                showRequestVolumes(value);
+                if (that._tileset) {
+                    that._tileset.debugShowViewerRequestVolume  = value;
+                }
+            }
+        });
+        /**
+         * Gets or sets the flag to show request volumes.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.showRequestVolumes = false;
+
+        var showGeometricError = knockout.observable();
+        knockout.defineProperty(this, 'showGeometricError', {
+            get : function() {
+                return showGeometricError();
+            },
+            set : function(value) {
+                showGeometricError(value);
+                if (that._tileset) {
+                    that._tileset.debugShowGeometricError = value;
+                }
+            }
+        });
+        /**
+         * Gets or sets the flag to show tile geometric error.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.showGeometricError = false;
+
+        var freezeFrame = knockout.observable();
+        knockout.defineProperty(this, 'freezeFrame', {
+            get : function() {
+                return freezeFrame();
+            },
+            set : function(value) {
+                freezeFrame(value);
+                if (that._tileset) {
+                    that._tileset.debugFreezeFrame = value;
+                    that._scene.debugShowFrustumPlanes = value;
+                }
+            }
+        });
+        /**
+         * Gets or sets the flag to suspend updates.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.freezeFrame = false;
+
+        var maximumSSE = knockout.observable();
+        knockout.defineProperty(this, 'maximumSSE', {
+            get : function() {
+                return maximumSSE();
+            },
+            set : function(value) {
+                maximumSSE(value);
+                if (that._tileset) {
+                    that._tileset.maximumScreenSpaceError = value;
+                }
+            }
+        });
+        /**
+         * Gets or sets the maximum screen space error.  This property is observable.
+         *
+         * @type {Number}
+         * @default 16
+         */
+        this.maximumSSE = 16;
```

Comment:
Any public methods and properties should spell out ScreenSpaceError instead of abbreviating to SSE. (I know you it was already like this, but this is a refactor of something that isn't even in master yet so we should fix it).